.github/workflows/pull-request.yml

```diff
@@ -57,14 +57,14 @@ jobs:
     - name: Install the package
       run: make install
 
-    - name: Code formatting
+    - name: Validate code formatting
       run: make lint
 
     - name: Commit code formatting changes
       run: |
         git add -u
         git commit -m "[skip ci] Code formatting" -a || true
-
+        

```

Comment:
The error is caused by pushing when there are no changes. Rather than checking for dependabot, I suggest skipping the last step when no changes are comitted